src/Feature/Catalog/code/ModelMappers/CatalogEntityMapper.cs

```diff
@@ -0,0 +1,35 @@
+﻿//    Copyright 2020 EPAM Systems, Inc.
+// 
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+// 
+//      http://www.apache.org/licenses/LICENSE-2.0
+// 
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+namespace Wooli.Feature.Catalog.ModelMappers
+{
+    using System.Reflection;
+
+    using AutoMapper;
+
+    using Foundation.DependencyInjection;
+
+    using Mapper = Foundation.Base.Mappers.Mapper;
+
+    [Service(typeof(ICatalogEntityMapper), Lifetime = Lifetime.Transient)]
+    public class CatalogEntityMapper : Mapper, ICatalogEntityMapper
+    {
+        public CatalogEntityMapper()
+        {
+            var configuration = new MapperConfiguration(cfg => cfg.AddProfiles(Assembly.GetExecutingAssembly()));
```

Comment:
You can specify mapping right here, because definitely there will be no more mappings in this class